Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -224,7 +224,7 @@ TextureCacheBase::TCacheEntryBase* TextureCacheBase::DoPartialTextureUpdates(Tex
 
 	TexCache::iterator iter = textures_by_address.lower_bound(entry_to_update->addr);
 	TexCache::iterator iterend = textures_by_address.upper_bound(entry_to_update->addr + entry_to_update->size_in_bytes);
-	bool entry_need_scaling = true;
+	bool entry_need_scaling = g_ActiveConfig.bCopyEFBScaled;
```

Comment:
Hmm, you mean check if the efb copy is upscaled, and then upscale the big texture as well? I really hope we can rely on all efb copies being upscaled, and when IR changes happen that all upscaled textures are removed from the texture cache. Otherwise we could run into a texture with scaling factor 3, which is to be updated with efb copies with scaling factors 1,2 and 4. That would be quite a mess to catch and require to rescale the texture, like from 2x to 3x.